public/app/features/alerting/unified/utils/alertmanager.test.ts

```diff
@@ -31,17 +31,11 @@ describe('Alertmanager utils', () => {
       });
     });
 
-    it('should parse escaped values correctly', () => {
-      expect(parseMatcher('foo=~"bar\\"baz\\""')).toEqual<Matcher>({
+    it('should not parse escaped values', () => {
```

Comment:
I'd document this test to explain _why_ it shouldn't be escaped by using some knowledge from https://github.com/prometheus/alertmanager/blob/4030e3670b359b8814aa8340ea1144f32b1f5ab3/pkg/labels/parse.go#L55-L99